core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/UniqueIDUserStoreManager.java

```diff
@@ -407,11 +407,11 @@ void updateUserListOfRoleWithID(String roleName, String[] deletedUserIDs, String
      * Get group details using group ID.
      *
      * @param groupID            Group ID.
-     * @param requiredAttributes Claims required.
+     * @param requestedClaims Claims required.
      * @return Group Object with details.
      * @throws UserStoreException If an error occurs while retrieving a group.
      */
-    Group getGroup(String groupID, List<String> requiredAttributes) throws UserStoreException;
+    Group getGroup(String groupID, List<String> requestedClaims) throws UserStoreException;
```

Comment:
I think we should have the same variable name through out. Either 'attributes' or 'claims'.